FluentUI.Demo/src/main/res/layout/activity_persistent_bottom_sheet.xml

```diff
@@ -16,7 +16,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:padding="@dimen/default_layout_margin"
-        android:background="?attr/fluentuiBackgroundColor"
+        android:background="?attr/fluentuiBottomSheetBackgroundColor"
```

Comment:
Why these changes?
I think, we should separate fixes in other PR
Just keep modularize code changes in this PR

Reply:
Corrected. The persistent bottomsheet will have many changes as it has been taken from modular branch which has @mnkshukl changes

Reply:
That is fine, but we should merge that change separately into develop first. So, that it does not create unwanted changes in this PR.
Request you to get those commits and raise a separate PR

Reply:
It will be cumbersome now  to do that. There were many changes in persistent bottom sheet and as per our earlier discussion, this was taken in to keep it consistent with modular persistent bottomsheet at once. @mnkshukl

Reply:
@aishwarya56 : As discussed, I compared the files of drawer module from modular branch to the ones in this PR. Most files are same, few changes are with respect to wrapping the context with the drawer theme
Another change is listitem has been moved from core to a separate module

Following is the list of changes 
- BottomSheetItemDivider.kt added themewrapper to constructor
- SheetHorizontalItemList.kt added themewrapper to constructor
- SheetHorizontalItemAdapter.kt passed Drawer theme to constructor
- SheetHorizontalItemView.kt added themewrapper to constructor
           - changed textAppearanceResId to R.style.TextAppearance_FluentUI_HorizontalListItemTitle from R.style.TextAppearance_FluentUI_ListItemSubtitle as listitem is now separate module
 - attrs.xml added fluentuiHorizontalListItemTitleColor
 - styles_font.xml, added TextAppearance.FluentUI.HorizontalListItemTitle
 - themes.xml added fluentuiHorizontalListItemTitleColor 
 - In values-night/themes.xml added section for BottomSheet which was missing in modular branch
 - fluentuiBottomSheetIconColor was added extra in night theme. removed now
 - In ListItemView.kt, added themewrapper to constructor
 - In ListSubHeaderView.kt added themewrapper to constructor